woocommerce/woocommerce
# Address CYS UI feedback 19 Sep

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fix two unresolved issues mentioned in PR  https://github.com/woocommerce/woocommerce/pull/40155
1. Unset the active state on color palette when user changes colors
2. Direct user back to the assembler hub automatically after saving



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use WP 6.3 
1. Install the `WooCommerce Beta Tester` plugin
1. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper and enable `customize-store` feature flag
1. Go to `WooCommerce > Home`
1. Click on `Customize your store` task 
1. Click on `Assembler Hub` button
1. Click on `Change the color palette`
1. Select a color palette
2. Click on `OR CREATE YOUR OWN`
3. Choose a background color
4. Observe that color palette active state should be unset
5. Click on save button
6. Should be redirected back to Assembler hub main screen.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix CYS UI issues

#### Comment 

</details>
